server/conf/database.properties.tmpl

```diff
@@ -9,10 +9,10 @@ scalar.db.username={{ default .Env.SCALAR_DB_USERNAME "" }}
 scalar.db.password={{ default .Env.SCALAR_DB_PASSWORD "" }}
 
 # Storage implementation. "cassandra" or "cosmos" or "dynamo" or "jdbc" or "grpc" can be set. Default storage is "cassandra".
-scalar.db.storage={{ default .Env.SCALAR_DB_STORAGE "cassandra" }}
+scalar.db.storage={{ default .Env.SCALAR_DB_STORAGE "" }}
```

Comment:
I removed default values because the default values are defined in the Docker file as environment variables.